test/signatures.test.ts

```diff
@@ -14,14 +14,25 @@ describe('getSignature', () => {
     expect(actual).toStrictEqual(expected);
   });
 
+  it('should return a correct scan result - uhash', async () => {
+    const fixturePath = path.join(__dirname, 'fixtures');
+    const filePath = path.join(fixturePath, 'example', 'main.cc');
+    const expected = {
+      path: filePath,
+      hashes_ffm: [{ format: 3, data: '15b464e1ed112524ebfae35b' }],
+    };
+    const actual = await getSignature(filePath, 'uhash');
+    expect(actual).toStrictEqual(expected);
```

Comment:
due of the suggestions here https://github.com/snyk/snyk-cpp-plugin/pull/38/commits/784a5e12fe941917c1527f309dd26d79401a17fe#r711707690

this would require all the tests to be changed to a format like this

```
    const expected = [
      {
        path: filePath,
        hashes_ffm: [{ format: 3, data: '15b464e1ed112524ebfae35b' }],
      },
    ];
    const actual = await getSignaturesByAlgorithm([filePath], 'uhash');
```

having as input filePaths array and output an array of fileSignatures